### PR TITLE
*(all): format doc comments to make linter happy on go1.19

### LINKED ISSUE #6708: Format doc comments to make linter happy on go1.19
### Is your feature request related to a problem?

Golang released 1.19 recently and the doc comments were refined (see: https://tip.golang.org/doc/go1.19#go-doc). The linter will report errors if we upgrade the compiler version to go1.19.

### Describe the feature you'd like

Adapt the latest version of the compiler tools.

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by lonng**
### What problem does this PR solve?

Issue Number: close #6708

### What is changed and how it works?

Golang released 1.19 recently and the doc comments were refined (see: https://tip.golang.org/doc/go1.19#go-doc). The linter will report errors if we upgrade the compiler version to go1.19.

This PR executes the `make check` based on the latest compiler.

#### Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/sink/mq/codec/craft/decoder.go [hunk 1]
```diff
@@ -25,12 +25,12 @@ import (
 	cerror "github.com/pingcap/tiflow/pkg/errors"
 )
 
-/// create string from byte slice without copying
+// / create string from byte slice without copying
```
Comment (overvenus) [inline cdc/sink/mq/codec/craft/decoder.go:28]: / is unnecessary

File: cdc/sink/mq/codec/craft/decoder.go [hunk 2]
```diff
@@ -25,12 +25,12 @@ import (
...
 )
 
-/// create string from byte slice without copying
+// create string from byte slice without copying
 func unsafeBytesToString(b []byte) string {
 	return *(*string)(unsafe.Pointer(&b))
 }
 
-/// Primitive type decoders
+// / Primitive type decoders
```
Comment (lonng) [inline cdc/sink/mq/codec/craft/decoder.go:33]:
```suggestion
// Primitive type decoders
```

File: engine/pkg/orm/client.go [hunk 3]
```diff
@@ -179,12 +179,12 @@ func (c *metaOpsClient) Close() error {
 	return nil
 }
 
-/////////////////////////////// Logic Epoch
+// ///////////////////////////// Logic Epoch
```
Comment (lonng) [inline engine/pkg/orm/client.go:182]:
```suggestion
// GenEpoch increases the backend epoch by 1 and return the new epoch
```

File: engine/pkg/orm/client.go [hunk 4]
```diff
@@ -179,12 +179,12 @@ func (c *metaOpsClient) Close() error {
...
 func (c *metaOpsClient) GenEpoch(ctx context.Context) (frameModel.Epoch, error) {
 	return c.epochClient.GenEpoch(ctx)
 }
 
-///////////////////////// Project Operation
+// /////////////////////// Project Operation
```
Comment (lonng) [inline engine/pkg/orm/client.go:187]:
```suggestion
//  Project Operation
```

File: engine/pkg/orm/client.go [hunk 5]
```diff
@@ -461,7 +461,7 @@ func (c *metaOpsClient) QueryWorkersByStatus(ctx context.Context, masterID strin
 	return workers, nil
 }
 
-/////////////////////////////// Resource Operation
+// ///////////////////////////// Resource Operation
```
Comment (lonng) [inline engine/pkg/orm/client.go:464]:
```suggestion
// Resource Operation
```

File: engine/pkg/orm/client.go [hunk 6]
```diff
@@ -376,7 +376,7 @@ func (c *metaOpsClient) QueryJobsByStatus(ctx context.Context,
 	return jobs, nil
 }
 
-/////////////////////////////// Worker Operation
+// ///////////////////////////// Worker Operation
```
Comment (lonng) [inline engine/pkg/orm/client.go:379]:
```suggestion
// Worker Operation
```

File: engine/pkg/orm/client.go [hunk 7]
```diff
@@ -276,7 +276,7 @@ func (c *metaOpsClient) QueryProjectOperationsByTimeRange(ctx context.Context,
 	return projectOps, nil
 }
 
-/////////////////////////////// Job Operation
+// ///////////////////////////// Job Operation
```
Comment (lonng) [inline engine/pkg/orm/client.go:279]:
```suggestion
// Job Operation
```

File: dm/master/server.go [hunk 8]
```diff
@@ -1739,22 +1739,28 @@ func extractWorkerError(result *pb.ProcessResult) error {
...
-//     * pause: related relay status is paused
-//     * resume: related relay status is running
+//
+//	OperateRelay:
+//	  * start: related relay status is running
+//	  * stop: related relay status can't be found in worker's result
+//	OperateWorkerRelay:
+//	  * pause: related relay status is paused
+//	  * resume: related relay status is running
+//
```
Comment (lance6716) [inline dm/master/server.go:1763]: seems markdown is not recgonized, I prefer

```
// Source:
//
//	OperateSource:
//	  - StartSource, UpdateSource: sourceID = Source
//	  - StopSource: return resp.Result = false && resp.Msg = “worker has not started”.
//
// Task:
//
//	StartTask, UpdateTask: query status and related subTask stage is running
//
//	OperateTask:
//	  - pause: related task status is paused
//	  - resume: related task status is running
//	  - stop: related task can't be found in worker's result
//
// Relay:
//
//	OperateRelay:
//	  - start: related relay status is running
//	  - stop: related relay status can't be found in worker's result
//
//	OperateWorkerRelay:
//	  - pause: related relay status is paused
//	  - resume: related relay status is running
//
```

File: dm/pkg/binlog/event/common.go [hunk 9]
```diff
@@ -33,14 +33,15 @@ type DDLDMLResult struct {
...
+//
 // for MariaDB:
-//   1. BinLogFileHeader, [ fe `bin` ]
-//   2. FormatDescriptionEvent
-//   3. MariadbGTIDListEvent, depends on genGTID
-//   -. MariadbBinlogCheckPointEvent, not added yet
+//  1. BinLogFileHeader, [ fe `bin` ]
+//  2. FormatDescriptionEvent
+//  3. MariadbGTIDListEvent, depends on genGTID
+//     -. MariadbBinlogCheckPointEvent, not added yet
```
Comment (lance6716) [inline dm/pkg/binlog/event/common.go:44]:
```suggestion
```

Now the format is wrong so we might delete it

File: dm/pkg/utils/db.go [hunk 10]
```diff
@@ -361,7 +361,7 @@ func GetGTIDMode(ctx context.Context, db *sql.DB) (string, error) {
 
 // ExtractTiDBVersion extract tidb's version
 // version format: "5.7.25-TiDB-v3.0.0-beta-211-g09beefbe0-dirty"
-//                               ^~~~~~~~~^
+// -                            ^..........
```
Comment (lance6716) [inline dm/pkg/utils/db.go:364]:
```suggestion
// to be extracted:              ^~~~~~~~~^
```

File: engine/jobmaster/dm/runtime/worker_status.go [hunk 11]
```diff
@@ -25,31 +25,34 @@ import (
 var HeartbeatInterval = 3 * time.Second
 
 // WorkerStage represents the stage of a worker.
-//          ,──────────────.      ,────────────.      ,─────────────.     ,──────────────.
-//          │WorkerCreating│      │WorkerOnline│      │WorkerOffline│     │WorkerFinished│
-//          `──────┬───────'      `─────┬──────'      `──────┬──────'     `──────┬───────'
-//                 │                    │                    │                   │
-//   CreateWorker  │                    │                    │                   │
+//
```
Comment (lance6716) [inline engine/jobmaster/dm/runtime/worker_status.go:28]: the alignment is lost

File: engine/servermaster/job_fsm.go [hunk 12]
```diff
@@ -41,42 +41,43 @@ type jobHolder struct {
...
-//     |<-----------------------------------------------|               |
-//     |                           |                    |               |
-//     | Master                    |                    |               |
-//     |  .OnWorkerDispatched      |                    |               |
-//     |  (with error)             |                    |               |
-//     |----------------------------------------------->|               |
-//     |                           |                    |               |
-//     |                           |                    |               |
-//     |                           |                    |               |
+//
```
Comment (lance6716) [inline engine/servermaster/job_fsm.go:44]: ditto

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: cdc/processor/pipeline/actor_node.go [hunk 13]
```diff
@@ -39,7 +39,9 @@ func NewActorNode(
 }
 
 // TryRun get message from parentNode and handle it util there is no more message to come
-//  or message handling is blocking
+//
```
Comment (overvenus) [inline cdc/processor/pipeline/actor_node.go:42]:
```suggestion
```

File: pkg/db/pebble.go [hunk 14]
```diff
@@ -198,7 +198,8 @@ func (p *pebbleDB) Close() error {
 }
 
 // TODO: Update metrics once we switch to pebble,
-//       as some metrics are not applicable to pebble.
+//
+//	as some metrics are not applicable to pebble.
```
Comment (overvenus) [inline pkg/db/pebble.go:202]:
```suggestion
// as some metrics are not applicable to pebble.
```

File: dm/pkg/utils/db.go (hunk 10 above)
Comment (lonng) [reply dm/pkg/utils/db.go:364]: It's a workaround for them. I tried a lot of ways and only this way was more accurately with our intention.

File: dm/master/server.go (hunk 8 above)
Comment (lonng) [reply dm/master/server.go:1763]: I change the code to make it close to your suggestion.

**Comment by lonng** [comment]
@lance6716 Comments addressed. PTAL again.

**Review by lance6716 — APPROVED** [review]

**Pull request merged**